src/iiiftilesource.js

```diff
@@ -181,20 +182,34 @@ $.extend( $.IIIFTileSource.prototype, $.TileSource.prototype, /** @lends OpenSea
     },
 
     /**
-     * Return the tileSize for the given level.
+     * Return the tileWidth for the given level.
      * @function
      * @param {Number} level
-    */
+     */
+    getTileWidth: function( level ) {
```

Comment:
@azaroth42 We're adding support for non-square tiles, so we're replacing tileSize with tileWidth and tileHeight. Could you take a quick look at these updated functions in the IIIFTileSource to make sure they've been translated properly?